### DISCUSSION TITLE
object array json response is "".

### DISCUSSION THREAD

**Discussion by no-miso** [mswjs/msw#1732] [Q&A] [answered]
hello, I'm trying to test using msw but something strange is happening.
The function that returns JSON for a single object works fine, but the function that returns JSON for an array object return data  `""`(empty string).
List other situation.

1: All functions fetch data from the same homebrew function used axios. Base URL has been set.
2: When I fetch without using a homebrew function and axios, msw returns the desired data, but it is not displayed.
3: If you start an API server with docker and fetch data, you will get data without any problems.
4: If you code the data as is without fetching it, you can display it without any problems.

successful handler
```
rest.get("http://localhost:8080/a", (req, res, ctx) =>
        const data: Info = {
            amount: 0
        };
        return res(ctx.status(200), ctx.json(data));
    })
```

failuerful handler
```
rest.get("http://localhost:8080/b", (req, res, ctx) => {
        const data: userInfo[] = [
            {
                id: '1',
                naming: 'foo',
                created_at: "2020-11-11"
            },
            {
                id: '2',
                naming: 'dog',
                created_at: "2020-11-11"
            }
        ];
        return res(ctx.status(200), ctx.json(data));
    })
```
Is there anything else I can do?
Thank you.

**Comment by no-miso** [comment, marked as answer]
The cause was that it was not written in handler.

**Reply by awhisler** [reply]
What is "it" and can you share what the fix of how to add "it" to the "handler"?
